src/ResourceManager/Security/Commands.Security.Test/ScenarioTests/SecurityAlertTests.ps1

```diff
@@ -0,0 +1,121 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Get policy events at management group scope
+#>
+function Get-AzureRmSecurityAlert-SubscriptionScope
+{
+    $alerts = Get-AzureRmSecurityAlert
+	Validate-Alerts $alerts
+}
+
+<#
+.SYNOPSIS
+Get policy events at management group scope
+#>
+function Get-AzureRmSecurityAlert-ResourceGroupScope
+{
+	$rgName = Get-TestResourceGroupName
+
+    $alerts = Get-AzureRmSecurityAlert -ResourceGroupName $rgName
+	Validate-Alerts $alerts
+}
+
+<#
+.SYNOPSIS
+Get policy events at management group scope
+#>
+function Get-AzureRmSecurityAlert-ResourceGroupLevelResource
+{
+	$rgName = Get-TestResourceGroupName
+
+    $alerts = Get-AzureRmSecurityAlert -ResourceGroupName $rgName -Location "westeurope" -Name "2518685747999999999_21d4f482-5538-4631-a7b6-7b345eda2954"
+	Validate-Alerts $alerts
+}
+
+<#
+.SYNOPSIS
+Get policy events at management group scope
+#>
+function Get-AzureRmSecurityAlert-SubscriptionLevelResource
+{
+    $alerts = Get-AzureRmSecurityAlert -Location "centralus" -Name "2518710774894070750_EEE23C70-80EF-4A8B-9122-507B0EA8DFFF"
+	Validate-Alerts $alerts
+}
+
+<#
+.SYNOPSIS
+Get policy events at management group scope
+#>
+function Get-AzureRmSecurityAlert-ResourceId
+{
+    $alerts = Get-AzureRmSecurityAlert -ResourceId "/subscriptions/487bb485-b5b0-471e-9c0d-10717612f869/resourceGroups/myService1/providers/Microsoft.Security/locations/westeurope/alerts/2518685747999999999_21d4f482-5538-4631-a7b6-7b345eda2954"
```

Comment:
as the previous comment, we can't create most of our resources on demand.
I removed as much "constants" as I could (if you see an opportunity to reduce even more I would like to know).
so instead of assuming a specific resource is there, I am now assuming that there a a resource of this type.
for example we have any alert in the tested subscription